.github/workflows/e2e_spark.yaml

```diff
@@ -10,7 +10,7 @@ on:
 jobs:
   k8s-integration-tests:
     name: "E2E about Spark Integration test"
-    runs-on: ubuntu-24.04
+    runs-on: ubuntu-20.04
```

Comment:
Does “ubuntu-22.04” not have this problem?
([#3556](https://github.com/volcano-sh/volcano/pull/3556) has been using ubuntu-22.04 before the change)

Reply:
spark related e2e is independent and use minikube as e2e tool，so it should be just consistent with upstream spark's e2e config.